src/AsyncWebSocket.cpp

```diff
@@ -876,6 +882,22 @@ void AsyncWebSocket::_handleDisconnect(AsyncWebSocketClient * client){
   });
 }
 
+bool AsyncWebSocket::availableForWriteAll(){
+  bool res = true;
+  for(const auto& c: _clients){
+    if(c->queueIsFull())res =false;
```

Comment:
It is possible here to return false immediately

Reply:
what you mean ?

Reply:
I mean you can write `return false;` instead of `res = false;`. That way you won't have to cycle through the remaining clients.

Reply:
Crap - what I did ? I should not stop drinking coffee these days...  anyway I have no excuse - this is not efficient code
Thank you for pointing it - I will redo PR properly

Reply:
redo https://github.com/me-no-dev/ESPAsyncWebServer/pull/410 and also changed condition to meet limit to prevent